[chore] [receiver/kafkametrics] Failing test TestIntegrationSingleNode

It was left uncaught because of https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/17037

See https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/3708080812/jobs/6285496856

```
Running target 'do-integration-tests-with-cover' in module 'receiver/kafkametricsreceiver' as part of group 'all'
make -C receiver/kafkametricsreceiver "do-integration-tests-with-cover"
make[1]: Entering directory '/home/runner/work/opentelemetry-collector-contrib/opentelemetry-collector-contrib/receiver/kafkametricsreceiver'
running go integration test ./... + coverage in /home/runner/work/opentelemetry-collector-contrib/opentelemetry-collector-contrib/receiver/kafkametricsreceiver
go test -race -timeout 360s -tags=integration,"" -run=Integration -coverprofile=integration-coverage.txt -covermode=atomic ./...
2022/12/15 22:03:17 github.com/testcontainers/testcontainers-go - Connected to docker: 
  Server Version: 20.10.21+azure-2
  API Version: 1.41
  Operating System: Ubuntu 22.04.1 LTS
  Total Memory: 6943 MB
2022/12/15 22:03:17 Starting container id: cf3d9be9ad9f image: docker.io/testcontainers/ryuk:0.3.4
2022/12/15 22:03:18 Waiting for container id cf3d9be9ad9f image: docker.io/testcontainers/ryuk:0.3.4
2022/12/15 22:03:18 Container is ready id: cf3d9be9ad9f image: docker.io/testcontainers/ryuk:0.3.4
2022/12/15 22:03:28 Starting container id: 88bd37fee83e image: johnnypark/kafka-zookeeper
2022/12/15 22:03:28 Waiting for container id 88bd37fee83e image: johnnypark/kafka-zookeeper
2022/12/15 22:03:32 Container is ready id: 88bd37fee83e image: johnnypark/kafka-zookeeper
--- FAIL: TestIntegrationSingleNode (50.31s)
    kafkametrics_e2e_test.go:101: waiting for metrics...
    kafkametrics_e2e_test.go:102: 
        	Error Trace:	/home/runner/work/opentelemetry-collector-contrib/opentelemetry-collector-contrib/receiver/kafkametricsreceiver/kafkametrics_e2e_test.go:102
        	Error:      	Condition never satisfied
        	Test:       	TestIntegrationSingleNode
        	Messages:   	expected metrics not received
FAIL
coverage: 20.6% of statements
FAIL	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/kafkametricsreceiver	50.375s
ok  	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/kafkametricsreceiver/internal/metadata	0.023s	coverage: 0.0% of statements [no tests to run]
FAIL
```

--- a/receiver/kafkametricsreceiver/kafkametrics_e2e_test.go
+++ b/receiver/kafkametricsreceiver/kafkametrics_e2e_test.go
@@ -52,6 +52,7 @@ func (h *testHost) ReportFatalError(err error) {
 var _ component.Host = (*testHost)(nil)
 
 func TestIntegrationSingleNode(t *testing.T) {
+	t.Skip("Skip failing test, see https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/17065")
 	ctx := context.Background()
 
 	req := testcontainers.ContainerRequest{

